corehq/apps/users/migrations/0024_add_login_as_all_users.py

```diff
@@ -8,12 +9,7 @@
 
 @skip_on_fresh_install
 def migrate_login_as_all_users_permission(apps, schema_editor):
-    roles = UserRole.view(
-        'users/roles_by_domain',
-        include_docs=False,
-        reduce=False
-    ).all()
-    for role_doc in iter_docs(UserRole.get_db(), [r['id'] for r in roles]):
+    for role_doc in iter_docs(UserRole.get_db(), get_all_role_ids()):
```

Comment:
I'm betting you could probably just make all these migrations noop at this point, same with the cleanup management commands.

Reply:
I did consider that but decided not to do it as part of this PR. I will likely have to do that as part of the SQL migration though.